docs/content/connectors/mysql-cdc.md

```diff
@@ -705,6 +705,17 @@ $ ./bin/flink run \
 ```
 **Note:** Please refer the doc [Restore the job from previous savepoint](https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/deployment/cli/#command-line-interface) for more details.
 
+### Tables Without primary keys
+
+In version 2.4.0, MySQL CDC support tables that do not have a primary key. To use a table without primary keys, you must configure the `scan.incremental.snapshot.chunk.key-column` option and specify one non-null field. 
```

Comment:
'Starting from version 2.4.0' should be more suitable, since it is available from 2.4.0 onwards